### DISCUSSION TITLE
Setting up Kubeapps from scratch - n00b - help please :)

### DISCUSSION THREAD

**Discussion by SpaceIgniter** [kubeapps/kubeapps#2919] [Q&A] [answered]
Hi Kubeapps contributors. Hope you are doing well.

I'm attempting to setup Kubeapps from scratch (including the prerequisites), but I'm not winning.
I've got an Ubuntu 20.04 VM on Oracle Cloud Infrastructure.
My main issue is installing the prerequisites. I've followed some guides/tutorials on Google without success.
Could anyone please link to a tutorial or give guidance on what I should do?

Kubeapps seems like a perfect platform for my intended use and I would sincerely like to use it and get it up and running.

Thank you for your time and assistance :)

**Comment by absoludity** [comment, marked as answer]
Hi @SpaceIgniter . Just to clarify, Kubeapps is only supported running on a Kubernetes cluster, not on an Ubuntu 20.04 VM. It's not clear to me whether you're actually using a Kubernetes cluster provided by Oracle Cloud Infrastructure, or thinking of using an Ubuntu 20.04 VM?

Our [Getting Started guide](https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md) is the normal place to which we point people, but as you're having issues installing the prerequisites (Helm? kubectl?), let us know which prerequisite is causing you pain :)

Hope that helps get you started!

**Reply by SpaceIgniter** [reply]
Hi,

Thank you for the response.
I confess my knowledge of the procedures and requirements is not sufficient.
I will attempt to setup the VM using an alternative method.
Since I don't use a Kubernetes cluster on Oracle, that seems like the cause.
Thank you for your clarification.

Enjoy your day ahead :)
